include/glaze/util/parse.hpp

```diff
@@ -124,22 +124,22 @@ namespace glz::detail
       }
    }
 
-   GLZ_ALWAYS_INLINE auto has_zero(const uint64_t chunk) noexcept
+   GLZ_ALWAYS_INLINE constexpr auto has_zero(const uint64_t chunk) noexcept
```

Comment:
@stephenberry 
just for reference constexpr implies inline on function declarations. Template functions are always inline, too, including those created with `auto`.

Reply:
Yeah, I was adding the constexpr specifier. But, I still keep GLZ_ALWAYS_INLINE, because this does force inlining on clang, so the compiler can't decide to not inline it, although in these small function cases I don't think it's necessary.

Reply:
Sounds great!